### PR TITLE
New sources and organizations

### PR DESCRIPTION
**Pull request by dessibelle**
### New sources

Act-On
AdAction
AdColony
Admarvel
Affluent
Aklamio
Applift
Bencom
Biano
BidTheatre
Button
Cake
ChineseAN.com
Cordial
Dable
DataLift360
Delta Projects Sting
Depo.hu
Digital Turbine
Emma
Feathr
Hardware.info
Homelook.it
Intent Media
Jobbird
Kieskeurig
LinkConnector
Meetrix Online
Mitula
Monetise
Najnakup
Pazaruvaj
Pepperjam Unity
Perform Media
Pricepanda
RecruitNow
Rejoiner
Salestring
Sanoma
Shopping-search.jp
Skapiec
SourceKnowledge
Sravni
Tapfiliate
Tapstone
Taptica
Taximail
The Affiliate Gateway
TimeOne Performance
Travel Supermarket
Twitter Organic
ValueCommerce
Vungle
We Love X

### New organizations

Act-On
Adaction
Adcolony
Aklamio
Applift
Bencom
Biano
BidTheatre
Button
Cake
ChineseAN.com
Cordial
Dable
Depo.hu
Digital Turbine
Emma
Feathr
Fluent
Hardware.info
Homelook.it
Intent Media
Jobbird
LinkConnector
Meetrix Online
Mitula
Monetise
Moneysupermarket.com
Najnakup
Pazaruvaj
Perform Media
Pricepanda
RecruitNow
Rejoiner
Reshift digital
Salestring
Sanoma
Shopping-search.jp
Skapiec
SourceKnowledge
Sravni
Tapfiliate
Tapstone
Taptica
Taximail
The Affiliate Gateway
TimeOne Performance
ValueCommerce
Vungle
We Love X

### Modified sources
Facebook Insights → Facebook Page Insights

### CODE REVIEW

File: organizations/reshift_digital.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/reshift_digital.json]: Is this supposed to be redshift digital?
org/icon?

File: organizations/theaffiliategateway.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/theaffiliategateway.json]: filename: the_affiliate_gateway.json

File: organizations/welovex.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/welovex.json]: filename: we_love_x.json

File: sources/datalift360.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "DATALIFT360",
+  "name": "DataLift360",
```
Comment (pfrisella) [inline sources/datalift360.json]:
```suggestion
  "name": "DataLift 360",
```

File: sources/datalift360.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/datalift360.json]: filename: datalift_360.json

File: sources/theaffiliategateway.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/theaffiliategateway.json]: filename: the_affiliate_gateway

File: sources/theaffiliategateway.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "THEAFFILIATEGATEWAY",
```
Comment (pfrisella) [inline sources/theaffiliategateway.json]:
```suggestion
  "id": "THE_AFFILIATE_GATEWAY",
```

File: sources/theaffiliategateway.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "The Affiliate Gateway",
+  "categories": ["ADVERTISING"],
+  "organization": "THEAFFILIATEGATEWAY",
```
Comment (pfrisella) [inline sources/theaffiliategateway.json]:
```suggestion
  "organization": "THE_AFFILIATE_GATEWAY",
```

File: sources/travelsupermarket.json [hunk 9]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "TRAVELSUPERMARKET",
+  "name": "Travel Supermarket",
```
Comment (pfrisella) [inline sources/travelsupermarket.json]:
```suggestion
  "name": "TravelSupermarket",
```

File: sources/welovex.json [hunk 10]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/welovex.json]: filename: we_love_x.json

File: sources/welovex.json [hunk 11]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "WELOVEX",
```
Comment (pfrisella) [inline sources/welovex.json]:
```suggestion
  "id": "WE_LOVE_X",
```

File: sources/welovex.json [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "We Love X",
+  "categories": ["ADVERTISING"],
+  "organization": "WELOVEX",
```
Comment (pfrisella) [inline sources/welovex.json]:
```suggestion
  "organization": "WE_LOVE_X",
```

File: organizations/theaffiliategateway.json [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "THEAFFILIATEGATEWAY",
```
Comment (pfrisella) [inline organizations/theaffiliategateway.json]:
```suggestion
  "id": "THE_AFFILIATE_GATEWAY",
```

File: organizations/welovex.json [hunk 14]
```diff
@@ -0,0 +1,6 @@
...
+  "id": "WELOVEX",
```
Comment (pfrisella) [inline organizations/welovex.json]:
```suggestion
  "id": "WE_LOVE_X",
```

File: sources/datalift360.json [hunk 15]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "DATALIFT360",
```
Comment (pfrisella) [inline sources/datalift360.json]:
```suggestion
  "id": "DATALIFT_360",
```

File: organizations/reshift_digital.json (hunk 1 above)
Comment (norric-funnel) [reply organizations/reshift_digital.json]: Nope, Reshift is the correct name. URL and icon fixed

**Comment by pfrisella** [comment]
Thanks for this contribution and for providing the list of sources/orgs in the PR description.

**Pull request merged**